core/processor/tx.go

```diff
@@ -168,20 +136,14 @@ func (t Tx) Command() txn.Command {
 }
 
 func (t Tx) GetPoWNonce() uint64 {
-	if t.version > 1 && t.pow != nil {
-		return t.pow.Nonce
-	}
-	return 0
+	return t.pow.Nonce
 }
```

Comment:
Dont you still want the nil check here in case of a validator command?

Reply:
The proof of work should be mandatory now. A transaction would be rejected without this. So there is no need to check that. However, I realize there is no verification on the commands for that. I will add it.

Reply:
Poof is not mandatory on validators commands.

Reply:
oh that change a lot of things

Reply:
It is troublesome to add the verification of the PoW right in the command verification. So I won't add it in this PR.

Reply:
ok.